pkg/cli/root.go

```diff
@@ -38,15 +38,19 @@ func Init() {
 // GetCommand returns the root command for the topo service
 func GetCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "topo {get,add,update,remove,watch} [args]",
+		Use:   "topo {get,add,update,remove,watch,add-entity,get-entity,add-relation,get-relation} [args]",
 		Short: "ONOS topology subsystem commands",
 	}
 
 	clilib.AddConfigFlags(cmd, defaultAddress)
 
 	cmd.AddCommand(clilib.GetConfigCommand())
 	cmd.AddCommand(getGetCommand())
+	cmd.AddCommand(getGetEntityCommand())
```

Comment:
To be more like other commands maybe get-entity and get-relationship should go in to the "get" command as "entity" and "relationship"

so the usage then would be like `onos topo get entity` etc.

same for add relationship and entity below

Reply:
Agreed.

Reply:
fixed